### ISSUE TITLE
Better handle the error on disk attachment.

### ISSUE THREAD

**Issue by aegiap** [Gandi/gandi.cli#108]
I tried to attach a disk to a VM and as the VM was in a bad state, the operation failed with the step error.

```
$ gandi disk attach sys_1425568864 myvm
Are you sure to attach disk 'sys_1425568864' to vm 'myvm' [y/N]: y
Attaching your disk(s).
Progress: [##############################################-----------------------] 66.67%  
00:00:19  Usage: gandi disk [OPTIONS] DISK VM

Error: step ERROR unknown, exiting
```
We should avoid printing a usage message in case of an operation in error.

More information:

```
$ cat /etc/debian_version 
8.0
```
```
$ gandi --version
Gandi CLI 0.12
```
```
$ dpkg -l gandicli
||/ Name                   Version          Architecture     Description
ii  gandicli               0.12             all              commandline client for Gandi.net
```

**Comment by rdoursenaud** [comment]
Same here, except the VM should have been in good shape since it was just created.

This prevents me from automating vm creation + disk creation/attachment. Annoying.
Is this an API issue ?

**Comment by rdoursenaud** [comment]
I have detected a pattern here. This seems to only happen when snapshots are activated.
Maybe the snapshot creation take a while but its state is not represented in the API making gandi.cli trying to mount an unfinished volume?

**Comment by sayoun** [comment]
Thanks for the report, do you have an operation ID so we can try to find on our side what may be the cause ? You can get it using `gandi oper list` after the attach fails in ERROR.

**Comment by rdoursenaud** [comment]
Nope, no operation ID.
See it fail at https://asciinema.org/a/18231

**Comment by rdoursenaud** [comment]
I can confirm that making it a two step process (create, wait a bit then attach) works a treat.
Will be my workaround for now.

**Comment by sayoun** [comment]
Thanks for the screencast, no operation indeed, I will try to reproduce this on our side

**Comment by sayoun** [comment]
Fixed in release 0.17 to have a better error message
